staff.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Staff
+---
+
+## Identifying Staff
+
+The _staff_ are the volunteers who act as network and system administrators to
+keep Libera.Chat running smoothly.
+
+The quickest way to find the list of current staff is to look for chanops among
```

Comment:
I believe the `/who libera/staff/*` command is more accurate for this than using chanops. For example `emerson` is (at this time) not in `#libera` and isn't in your list but is a staff member.

Reply:
Fixed

Reply:
There's also ozone (Sigyn) which is opped but not a staff member, yeah.